CMakeLists.txt

```diff
@@ -67,6 +67,13 @@ add_definitions(
         -DVCS_REVISION_ID=0x${vcs_revision_id}ULL
         -DNODE_NAME="org.opencyphal.ocvsmd"
 )
+if (DEFINED PLATFORM_LINUX_TYPE)
+    if(${PLATFORM_LINUX_TYPE} STREQUAL "bsd")
+        add_definitions(-DPLATFORM_LINUX_TYPE_BSD)
+    elseif(${PLATFORM_LINUX_TYPE} STREQUAL "debian")
+        add_definitions(-DPLATFORM_LINUX_TYPE_DEBIAN)
```

Comment:
Debian comes with BSD kernels as well, not just Linux. While we are at it, Linux is a kernel, not an OS, that would be GNU/Linux. Should we not simply go like `PLATFORM_DEBIAN`, `PLATFORM_BSD`? Although then it seems odd that one of them is an OS while the other is a kernel.